build.gradle

```diff
@@ -1,3 +1,10 @@
+plugins {
+    id 'java'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
+    id 'signing'
+    id 'maven'
+}
+
```

Comment:
If you're going to add this, I think you should remove the plugin declarations from line 20 onwards.

Reply:
This was caused by a bad local revert. The legacy plugin stuff is removed now.